packages/design-system/src/Checkbox/Checkbox.styles.tsx

```diff
@@ -0,0 +1,165 @@
+import styled, { css } from "styled-components";
+
+const Variables = css`
+  --checkbox-color-label: var(--ads-v2-color-fg);
+  --checkbox-color-border: var(--ads-v2-color-border);
+  --checkbox-color-background-selected: var(--ads-v2-color-bg-brand-secondary);
```

Comment:
Let's use a single variable for one element - background, or border - and manipulate it within the styles of the class. This way we are not increasing the surface area of our component, and we are following the same pattern for css variable manipulation across components.